gym/f110_gym/envs/f110_env.py

```diff
@@ -354,12 +435,13 @@ def render(self, mode='human'):
             # first call, initialize everything
             from f110_gym.envs.rendering import EnvRenderer
             self.renderer = EnvRenderer(WINDOW_W, WINDOW_H)
-            self.renderer.update_map(self.map_name, self.map_ext)
+            map_img_path = os.path.dirname(os.path.abspath(__file__)) + '/maps/' + self.map_name
```

Comment:
Good catch, this wouldn't work for custom maps when the full path is given. I think using self.map_path and strip the .yaml extension would work.

Reply:
Yeah, I agree.
It didn't work for one of my maps, that is how I found it.